bin/crew

```diff
@@ -115,6 +115,11 @@ at_exit do
   ExitMessage.print
 end
 
+def crewlog
```

Comment:
Would it be better to pass the message as a parameter?  I know this will require the message to be created outside the function but it would make it more flexible.  Right now, this generic named function is only logging for is_fake.

Reply:
I only have this in there for messing around to figure out what is going on. I will remove it later.